server/test/services/navigation/definition.ts

```diff
@@ -267,7 +267,7 @@ describe("Parser", () => {
             { line: 3, character: 10 },
             {
               start: { line: 1, character: 0 },
-              end: { line: 6, character: 0 },
```

Comment:
What about the parser updates forced the test change?

Reply:
Looks like the parsing output changed by 1 line, in this case going from an import path to a contract. I assume it's SourceUnit because it includes from the pragma statement down to the last line of the file. It misses the last line, which is the closing bracket. I don't think this can meaningfully impact users. All the other jumps seem to work fine (jump to definitions, implementations, references etc). It's just the jump to imported file which changed

Reply:
Okay cool.